generic/ltp.py.data/ltp.yaml

```diff
@@ -0,0 +1,58 @@
+setup:
```

Comment:
Btw I'm wondering why those prefixes. I'd simply put all params in global so one could inject this file anywhere...